### PR TITLE
AWS doc fixes and auth doc additions

### PR DESCRIPTION
**Pull request by dwhitena**
Fixes: https://github.com/pachyderm/pachyderm/issues/2458, https://github.com/pachyderm/pachyderm/issues/2454, and https://github.com/pachyderm/pachyderm/issues/2415

### CODE REVIEW

File: doc/deployment/amazon_web_services.md [hunk 1]
```diff
@@ -223,10 +225,11 @@ pachd               1.6.0
 
 ## One Shot Script
 
-### Install additional prerequisites
-
-This scripted deploy requires a couple of prerequisites in addition to the ones listed under [Prerequisites](amazon_web_services.md#prerequisites):
+### Prerequisites
 
+- [AWS CLI](https://aws.amazon.com/cli/) - have it installed and have your [AWS credentials](http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) configured.
```
Comment (JoeyZwicker) [inline doc/deployment/amazon_web_services.md]: We should add pachctl as a prereq. Users often forget that.

Comment (dwhitena) [reply doc/deployment/amazon_web_services.md]: It's included explicitly here as part of the Pachyderm installation: https://github.com/pachyderm/pachyderm/blob/b593fc2c567d9dc4310e6cf6ea495ebf1b430db8/doc/deployment/amazon_web_services.md#install-pachctl. But I'm fine with moving it anywhere. If we move it up to the top, we should do that in all of the deploy docs and then maybe just have a common doc somewhere that talks about installing pachctl.

**Comment by JoeyZwicker** [comment]
I know it's its own section, I'm simple suggesting adding an "install pachctl" bullet point to all the prereq lists so we nip that one in the bud

**Comment by dwhitena** [comment]
@JoeyZwicker Updated. Also, at least partially, fixed some confusion from: https://github.com/pachyderm/pachyderm/issues/2254

**Pull request merged**